app/views/shared/_links_navbar.html.haml

```diff
@@ -1,12 +1,27 @@
-%nav      
-    = link_to_unless_current t('english'), locale: "en"
-    = link_to_unless_current t('svenska'), locale: "se" 
-    = link_to 'D-pressed', root_path  
-    - unless current_user
-        = link_to t('registration'), new_user_registration_path
-        = link_to t('log_in'), new_user_session_path
-    - if user_signed_in?   
-        = link_to t('my_account'), user_path(current_user)
-        = link_to t('log_out'), destroy_user_session_path, method: :delete
-        - if current_user.author? || current_user.editor?
-            = link_to t('new_article'), new_article_path
\ No newline at end of file
+%nav.bg-grey.text-black.py-5
+    .notice
+        = flash[:notice]
+    .alert
+        = flash[:error]
+    .container.md:flex.md:items-center.md:justify-between.mx-auto.px-5
+        %h1.mb-4.md:mb-0.mx-auto.text-2xl= link_to 'D-pressed', root_path 
+        %ul.md:flex.md:items-center.list-reset.text-xl
+            %li.border-t.md:border-0.md:ml-4
+                - unless current_user
+                    %a.block.md:inline.no-underline.py-4.px-4.md:py-0.text-white.hover:text-grey-lighter   
```

Comment:
this should also be moved to the css file.

Reply:
replace the line with a class name ( I used `.links` ) and then in the css file
```
.links {
  @apply block no-underline py-4 px-4 text-white;
}

.links:hover {
  @apply text-grey-lighter;
}

@screen md {
  .field {
    @apply text-center mb-0;
  }

  .links {
    @apply inline py-0;
  }
}
```

Reply:
I will remove this Tailwind code completely and deal with the styling of the links in the application.css on its own.